packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -74,7 +74,7 @@ export class EpochTracker implements IPersistedFileCache {
         // Limits the max number of concurrent requests to 24.
         this.rateLimiter = new RateLimiter(24);
 
-        // We need this for GC testing until we properly plumb through the snapshot expiration policy (see PR #11168)
+        // Matches the TestOverride logic for the policy defined in odspDocumentStorageServiceBase.ts
         this.snapshotCacheExpiryTimeoutMs =
             loggerToMonitoringContext(logger).config.getBoolean("Fluid.Driver.Odsp.TestOverride.DisableSnapshotCache")
```

Comment:
Would there be value in actually testing a snapshot cache that is non 0 maybe 5-10 seconds or less?

Reply:
Hmm good question.  The only difference that brings from just using 0 is if you refresh the page twice in 5 seconds, right?  I don't think it's an interesting scenario if I'm right.

But I do see your point... all this cache expiry stuff is kind of theoretical and hard to test in practice.  Maybe for stress tests we would want a meaningful cache expiry value.

That can be a follow-on change in main someday to support that kind of test, so not making a change now.  I'm not too worried about compat for TestOverrides so we can just make it work when the time comes.  I'll make a note about it though.